pkgs/development/python-modules/pyisbn/default.nix

```
@@ -0,0 +1,22 @@
+{ buildPythonPackage, fetchPypi, hypothesis, lib, pytest, pytest-randomly
+, pytest-cov }:
+
+buildPythonPackage rec {
+  pname = "pyisbn";
+  version = "1.3.1";
+
```

Only [>=3.6](https://github.com/JNRowe/pyisbn/blob/main/setup.cfg#L62) is support.

```suggestion
  version = "1.3.1";
  format = "setuptools";

  disabled = pythonOlder "3.7";
```